gnovm/stdlibs/std/native.go

```diff
@@ -27,6 +27,10 @@ func GetChainID(m *gno.Machine) string {
 	return GetContext(m).ChainID
 }
 
+func GetChainDomain(m *gno.Machine) string {
+	return GetContext(m).ChainDomain
```

Comment:
Since this is used to write the import paths, should this not be largely known by gno files already (so no need for the function)?

Reply:
It's for when your `p/` package is imported from one chain to another. The code remains the same, but the behavior may change.